scripts/splash/index.tsx

```diff
@@ -202,7 +202,7 @@ const App = () => {
           <Color dim>
             <Box width={3}>💡</Box>
             <Box>
-              Tip: to disable these reports, run{' '}
```

Comment:
This causes a regression where a space is missing between `run` and `DISABLE_SPLASH`:

![Screen Shot 2019-11-08 at 3 20 51 PM](https://user-images.githubusercontent.com/85783/68517195-65a72400-023b-11ea-8b16-c1b5ef953abc.png)